learn/tanssi/appchain-services/block-production.md

```diff
@@ -84,7 +84,7 @@ Every new assignment works intentionally with a one-session delay, so the sequen
 
 When a new session starts, the Tanssi protocol will put the queued assignment into effect. Sequencers will automatically change and start producing blocks in the new Tanssi appchain they've been assigned to while discarding the chain state from the previous assignment. Tanssi will also calculate the new assignment, considering changes in Tanssi appchains that might have been activated or deactivated and block producers that might have been added or removed from the pool or changed the total staked value. This new assignment will be queued for the next session.
 
-![Sessions](/images/learn/tanssi/technical/technical-2.webp)
+![Sessions](/images/learn/tanssi/appchain-services/block-production/block-production-1.webp)
```

Comment:
The problem with his image is that it has the relay chain, which says Tanssi appchains, right?

Reply:
No problems with this image. In fact, it didn't change, it was just renamed for consistency with the new structure

Reply:
Ahhhhh I had the wrong image. looks great!